judgments/forms/search_forms.py

```diff
@@ -35,6 +35,24 @@ def _get_choices_by_group(courts):
 TRIBUNAL_CHOICES = _get_choices_by_group(all_courts.get_grouped_selectable_tribunals())
 
 
+def get_short_identifiers(top_level_choices):
+    all_shorts = set()
+    for key, value in top_level_choices:
+        if isinstance(value, list):
+            all_shorts = all_shorts | set(x[0].partition("/")[0] for x in value)
+    return all_shorts
```

Comment:
What is value in this instance? it's hard to figure out if `set(x[0].partition("/")[0] for x in value)` might fail under certain types of data. If it's typed it might tell you that there could be a potential time when accessing [0] on x would fail, or partition could fail

Reply:
`top_level_choices` is essentially a list of key-value pairs as a list; [('ewca/civ', 'Civil Division'), ('ewca/crim', 'Criminal Division')]), coming out of a dictionary, so x[0] is almost certainly valid. As long as it's a string, partition("/") will always return three things, the first being "all the text before the `/`", even if there isn't one.

Reply:
Okay cool, thanks for explaining